components/notebook-controller/config/overlays/openshift/odh-metadata.yaml

```diff
@@ -0,0 +1,4 @@
+releases: 
```

Comment:
Please move this file, two directories back
https://github.com/opendatahub-io/kubeflow/tree/main/components/notebook-controller/config

as shared in JIRA and slack: https://redhat-internal.slack.com/archives/C060A5FJEAD/p1732213663369219